pkg/services/apps/apps_test.go

```diff
@@ -112,6 +112,54 @@ func TestGetOne(t *testing.T) {
 	}
 }
 
+func TestGetUsers(t *testing.T) {
+	tests := map[string]struct {
+		id               int32
+		expectedResponse []AppUser
+		expectedError    error
+		repository       *test.MockRepository
+	}{
+		"it gets users for an app": {
+			id: 1,
+			expectedResponse: []AppUser{{
+				ID:        oltypes.Int32(1),
+				Firstname: oltypes.String("First"),
+				Lastname:  oltypes.String("Last"),
+				Username:  oltypes.String("User Name"),
+				Email:     oltypes.String("E-Mail"),
+			}},
+			expectedError: nil,
+			repository: &test.MockRepository{
+				ReadFunc: func(r interface{}) ([][]byte, error) {
+					b, err := json.Marshal([]AppUser{{
+						ID:        oltypes.Int32(1),
```

Comment:
This is a bit scary 😅 

Is this how it's done everywhere else? We've hit integer rollover before.